components/expenses/CreateExpenseForm.js

```diff
@@ -385,13 +385,13 @@ class CreateExpenseForm extends React.Component {
 
             <div className="col">
               <label>
-                <FormattedMessage id="expense.amount" defaultMessage="amount" />
+                <FormattedMessage id="expense.amount.dollars" defaultMessage="amount in US dollar" />
               </label>
               <div className="amountDetails">
                 <span className="amount">
                   <InputField
                     defaultValue={expense.amount}
-                    pre={getCurrencySymbol(collective.currency)}
+                    pre={`US${getCurrencySymbol(collective.currency)}`}
```

Comment:
Collectives can have different currencies than US dollars, that's why we do `getCurrencySymbol`. We should not enforce `US` here.

Reply:
![image](https://user-images.githubusercontent.com/30627428/62477494-00a5e300-b7a1-11e9-835e-c8badedca8ec.png)

Reply:
@Betree have you reviewed this?